.teamcity/requirements.txt

```diff
@@ -3,3 +3,7 @@ paddlepaddle-gpu==1.3.0.post97
 gym
 details
 parameterized
+
+# requirements for compiling the docs
```

Comment:
requirements are unnecessary?

Reply:
Oh, I forgot the remove these lines.
I had installed all the dependency libraries in the anaconda environment.